pages/algorithms/exposed-to-assistive-technologies.md

```diff
@@ -1,5 +1,8 @@
 ---
 title: Exposed to assistive technologies
+key: exposed-to-assistive-technologies
 ---
 
-Elements should be contained in the accessibility tree as described in [Core Accessibility API Mappings 1.1](https://www.w3.org/TR/core-aam-1.1/#mapping_general)
+Elements contained in the accessibility tree as described in [Core Accessibility API Mappings 1.1](https://www.w3.org/TR/core-aam-1.1/#mapping_general) and associated native markup language API mappings, e.g. [HTML Accessibility API Mappings](https://www.w3.org/TR/html-aam) and [SVG Accessibility API Mappings](https://www.w3.org/TR/svg-aam).
```

Comment:
This part is open ended:
> and associated native markup language API mappings, e.g. ...

This definition allows other API mappings in addition to HTML and SVG. That leaves room for gaps and differences between implementations. I suggest we limit this to HTML and SVG - since there are no others at the moment. If that changes at some point in the future, we can simply update the definition. As it is, this definition wouldn't be permitted in an applicability section of a rule, which seems to be the goal here.